.github/scripts/workflow-monitor.py

```diff
@@ -36,11 +36,12 @@ def main(workflow_id, gha_ci_personal_token):
     state = None
     consecutive_failures = 0
     headers = {'Authorization': "token {}".format(gha_ci_personal_token.strip())}
+    gha_workflow_api_url = f"{gha_runs_api_url}/{workflow_id}"
```

Comment:
Nit: I would just inline this into L44

Reply:
I disagree, headers is already hoisted up here.  This seems fine.

Reply:
In other locations, `headers` is used in multiple places and thus is separated out (for ex:https://github.com/firesim/firesim/blob/b157da0b7ba889bf14d4a122aa04b044e849597a/.github/scripts/common.py#L87). Here ideally I think both `headers` and `gha_workflow_api_url` would be inlined (if they are used once inline). But like I said "nit" - very low priority.